baremetal/metal3machine_manager_test.go

```diff
@@ -618,8 +618,8 @@ var _ = Describe("Metal3Machine manager", func() {
 
 		DescribeTable("Test ChooseHost",
 			func(tc testCaseChooseHost) {
-				c := fakeclient.NewClientBuilder().WithScheme(setupScheme()).WithObjects(tc.Hosts...).Build()
-				machineMgr, err := NewMachineManager(c, nil, nil, tc.Machine,
+				fakeClient := fakeclient.NewClientBuilder().WithScheme(setupScheme()).WithObjects(tc.Hosts...).Build()
```

Comment:
It looks a bit dangerous to me to have both `fakeclient` and `fakeClient`. In some other places I think we have just `fake` for the import. What do you think about doing the same here?
```suggestion
				fakeClient := fake.NewClientBuilder().WithScheme(setupScheme()).WithObjects(tc.Hosts...).Build()
```